middleware/camera/onvif_zeep_camera_controller.py

```diff
@@ -0,0 +1,144 @@
+import logging
+from django.conf import settings
+from onvif import ONVIFCamera, ONVIFError
+from middleware.camera.abstract_camera import AbstractCameraController
+from middleware.camera.exceptions import InvalidCameraCrendentialsException
+from middleware.camera.types import CameraAsset
+from middleware.camera.utils import wait_for_movement_completion
+
+logger = logging.getLogger(__name__)
+
+
+# my_cam=ONVIFCamera("192.168.68.116", 80, "remote_user","2jCkrCRSeahzKEU", "/home/teleicu/.local/share/virtualenvs/teleicu_middleware_backend-pwPF-mjP/lib/python3.12/site-packages/wsdl/")
+class OnvifZeepCameraController(AbstractCameraController):
+
+    def __init__(self, req: CameraAsset) -> None:
+        try:
+            # cam = ONVIFCamera(
+            #     req.hostname, req.port, req.username, req.password, settings.WSDL_PATH
+            # )
```

Comment:
```suggestion
```

Reply:
hey i wanted to ask you this so WSDL file path is optional and we will have to see where it is installed in the docker directory so i had kept this as a comment 
if we pass this file it confirms that the camera meets all the onvif standards (something on these lines)
so its option and everything works even without it so wanted to know should we consider this or ignore it ?

Reply:
in settings try to resolve the module path 

```py
from pathlib import Path
import onvif
wsdl_path = Path(onvif.__file__).parent.parent / "wsdl"
```

Reply:
have resolved it